Dockerfile

```diff
@@ -12,13 +12,11 @@
 WORKDIR /app
 
 ENV PYTHONUNBUFFERED=1
+ENV ENV_ENGINE="default"
+ENV ENV_NAME="default"
+ENV ENV_USER="default"
+ENV ENV_PASSWORD="default"
```

Comment:
The environment variable `ENV_PASSWORD` is set to 'default'. It's important to avoid hardcoding sensitive information like passwords. Consider using Docker secrets or environment variables set at runtime to manage sensitive data securely.